packages/server/src/api/routes/user.js

```diff
@@ -34,5 +34,10 @@ router
     authorized(PermissionTypes.USER, PermissionLevels.WRITE),
     controller.destroyMetadata
   )
+  .post(
+    "/api/users/sync/:id",
```

Comment:
Should it be `/api/users/metadata/sync/:id`? to be consistent with the rest of the routes in the api